Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -78,6 +78,7 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
              --resource-group "${AZURE_RESOURCE_GROUP}" \
              --name "${NAME}" \
              --network-plugin none
+             --generate-ssh-keys
```

Comment:
Copy-pasting this snippet will fail, because the `--generate-ssh-keys` will be considered a separate command. You're missing a backslash on the previous line to escape the line break. Please fix it

Reply:
Hi, I'll create a new Pull Request following your recommendations.
Thank you for reviewing it!

Reply:
@qmonnet Hi Quentin, I just opened the new PR (#35270) and included everything you recommended.

Reply:
Thanks! Please try to update a PR (update your commit locally, then force-push to your branch) rather than opening a new PR, next time :). I'm happy to help if necessary.

Reply:
It was the only way I knew how to do it haha, I'm new to this contributor thing.
Thanks for the help!!!